gaphor/UML/toolbox.py

```diff
@@ -19,3 +25,16 @@
     use_cases,
     profiles,
 )
+
+uml_diagram_types: DiagramTypes = (
+    DiagramType("cls", gettext("New Class Diagram"), (classes,)),
+    DiagramType("pkg", gettext("New Package Diagram"), (classes,)),
+    DiagramType("cmp", gettext("New Component Diagram"), (components,)),
+    DiagramType("dep", gettext("New Deployment Diagram"), (components,)),
+    DiagramType("act", gettext("New Activity Diagram"), (actions,)),
+    DiagramType("sd", gettext("New Sequence Diagram"), (interactions,)),
+    DiagramType("com", gettext("New Communication Diagram"), (interactions,)),
+    DiagramType("stm", gettext("New State Machine Diagram"), (states,)),
+    DiagramType("uc", gettext("New Use Case Diagram"), (use_cases,)),
+    DiagramType("prf", gettext("New Profile Diagram"), (profiles,)),
```

Comment:
I don't think UML has an abbreviation for Class Diagrams or Profile Diagrams, and I think many of the tools just write it out. I think it makes sense, though, using an abbreviation for them like you did :+1:

Reply:
It's kinda weird to have a full name, where in all other diagram types an abbreviation is used.